src/main/java/com/redhat/cloud/notifications/GwResource.java

```diff
@@ -51,13 +48,13 @@
     @Channel("egress")
     Emitter<String> emitter;
 
-    @Inject
-    @Metric(name = "notifications.gw.received")
-    Counter receivedActions;
+    private final Counter receivedActions;
+    private final Counter forwardedActions;
 
-    @Inject
-    @Metric(name = "notifications.gw.forwarded")
-    Counter forwardedActions;
+    public GwResource(MeterRegistry registry) {
+        this.receivedActions = Counter.builder("notifications.gw.received").register(registry);
+        this.forwardedActions = Counter.builder("notifications.gw.forwarded").register(registry);
```

Comment:
This will work, but can we create the counters in a similar fashion in all projects?
```suggestion
        receivedActions = registry.counter("notifications.gw.received");
        forwardedActions = registry.counter("notifications.gw.forwarded");
```